### PR TITLE
release: Update release notes with GitHub issues

### PR DESCRIPTION
**Pull request by carlescufi**
List all issues closed after the date of the 1.10 release (8th December
2017).

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

**Comment by carlescufi** [comment]
Note, I listed **all** the issues, including those that were closed as duplicates.

**Comment by carlescufi** [comment]
List obtained as per; https://github.com/zephyrproject-rtos/zephyr/wiki/Release-procedure#listing-all-closed-github-issues

File: doc/release-notes-1.11.rst [hunk 1]
```diff
@@ -184,6 +184,322 @@ Issue Related Items
...
+* :github:`4597` - tests/drivers/ipm/testcase.yaml#test :unable to print the Expected output
+* :github:`4603` - sanitycheck either conceals information from user or spams it
+* :github:`4606` - usb mass storage : config waits for Vendor ID and Product ID from user during building
+* :github:`4633` - Implement flash page layout api in the Kinetis flash driver
+* :github:`4635` - xtensa-esp32-elf-gcc.exe: error: unrecognized command line option '-no-pie'
+* :github:`4653` - net: tcp->recv_max_ack isn't used
+* :github:`4666` - x86 device trees need interrupt controller nodes
+* :github:`4687` - Windows: Remove all dependencies on MSYS2
+* :github:`4699` - PWM LED Driver for ESP32
+* :github:`4705` -  tests/net/socket/tcp/ :undefined reference to `__getreent
```
Comment (dbkinder) [inline doc/release-notes-1.11.rst]: this line gives a doc gen error, remove the `` ` ``

File: doc/release-notes-1.11.rst [hunk 2]
```diff
@@ -184,6 +184,322 @@ Issue Related Items
...
+* :github:`5762` - Windows 10 WSL does not supports Native POSIX Boards
+* :github:`5766` - boards: nucleo_f413zh: Likely outdated OpenOCD info in docs
+* :github:`5771` - Linking issues with host cross compile with cmake
+* :github:`5772` - sanitycheck crashes if ZEPHYR_BASE has symlinks in its path
+* :github:`5778` - Add/fix flash controller nodes for NXP kinetis SoCs
+* :github:`5779` - bluetooth test_controller_4_0 fails to build on nrf52840_pca10056
+* :github:`5784` - make rom_report fails for qemu_x86 (not finding zephyr.bin)
+* :github:`5794` - wiki/Development-Model is out of date
+* :github:`5808` - msys2 getting started instructions are missing Ninja install step
+* :github:`5817` - socket.h: Using #define for POSIX redefinition of zsock_ functions has unintended consequences
```
Comment (dbkinder) [inline doc/release-notes-1.11.rst]: Sphinx thinks the zsock_ is a link reference,  change this to ``zsock\_``

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Comment by carlescufi** [comment]
@dbkinder fixed

**Comment by dbkinder** [comment]
missed one (line 302)

**Comment by carlescufi** [comment]
@dbkinder fixed

**Comment by carlescufi** [comment]
@dbkinder CI passes now

**Review by dbkinder — APPROVED** [review]
yea!

**Pull request merged**